packages/client/src/pages/Game/components/Header/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+
+import './styles.scss';
+
+export const Header: React.FC = () => {
+  const [score, setScore] = useState(340);
+  const [bestScore, setBestScore] = useState(340);
+
+  const startNewGame = () => {
+    setScore(0);
+  };
+
+  return (
+    <div className="header">
+      <div className="header__block">
+        <span className="header__block_title">
```

Comment:
По БЭМ-у, block это элемент header-а, а title элемент2 block__elem1__elem2
```suggestion
      <div className="header__block">
        <span className="header__block__title">
```